### DISCUSSION TITLE
一些好用的 Mac 软件 - Alan|前端博客

### DISCUSSION THREAD

**Discussion by giscus** [3Alan/site#26] [Announcements]
# 一些好用的 Mac 软件 - Alan|前端博客

收藏了一些好用的 Mac 软件，包含录屏，浏览器

https://www.alanwang.site/blog/mac-app

**Comment by Ryokki** [comment]
可以给我分享一个Arc的邀请码吗

**Reply by 3Alan** [reply]
https://arc.net/gift/ca47d9d3
